docs/DependencyInjection.md

```diff
@@ -283,4 +285,4 @@ When registering a FusionCache instance via the builder we always had the abilit
 
 This is available for all the building blocks: logger, memory cache, memory locker, distributed cache, serializer, backplane and plugins.
 
-Nice.
\ No newline at end of file
+Nice.
```

Comment:
Something minor and non-blocking, but I'm not understanding these diffs:

![image](https://github.com/user-attachments/assets/f9c67d77-68f6-41c2-bf61-bb1115eb5c3a)

Maybe it's just a newline thing? Or an encoding thing? But then, why only on the last line?